### PR TITLE
Do 138 add CustomPagination 

### PR DESCRIPTION
**Pull request by Baranovsky1986**
add CustomPagination

### CODE REVIEW

File: data_converter/pagination.py [hunk 1]
```diff
@@ -8,15 +9,13 @@ class CustomPagination(PageNumberPagination):
     page = DEFAULT_PAGE
```
Comment (roman-tiukh) [inline data_converter/pagination.py:9]: what is the purpose of the variable `DEFAULT_PAGE`?

File: data_converter/pagination.py [hunk 2]
```diff
@@ -8,15 +9,13 @@ class CustomPagination(PageNumberPagination):
...
-            },
-            # 'total': self.page.paginator.count,
-            'page': int(self.request.GET.get('page', DEFAULT_PAGE)),
-            'page_size': int(self.request.GET.get('page_size', self.page_size)),
-            'results': data
+            'count': self.page.paginator.count,
+            'last_page': math.ceil(self.request.GET.get(
+                'last_page',
+                self.page.paginator.count/int(self.request.GET.get('page_size', self.page_size))
+            )),
```
Comment (roman-tiukh) [inline data_converter/pagination.py:19]: why `self.request.GET.get` here? at 16 line

Comment (roman-tiukh) [reply data_converter/pagination.py:19]: hardcode page_size you can replace with class variable

File: data_converter/pagination.py (hunk 1 above)
Comment (Baranovsky1986) [reply data_converter/pagination.py]: deleted it

File: data_converter/pagination.py (hunk 2 above)
Comment (Baranovsky1986) [reply data_converter/pagination.py]: fixed according your comments

**Pull request merged**